dev-cpp/gsl/gsl-0_pre20171105.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Guideline Support Library implementation by Microsoft."
+HOMEPAGE="https://github.com/Microsoft/GSL"
+SRC_URI="https://github.com/Microsoft/GSL/archive/master.zip -> ${P}.zip"
```

Comment:
Every time they make a commit, the master.zip archive will change. That's no good.
Consider referencing a tag (ideal) or a commit (if you must). They don't have any tags, so a commit is fine: https://github.com/Microsoft/GSL/archive/d10ebc6555b627c9d1196076a78467e7be505987.tar.gz (for example)